qemu/tests/qemu_guest_agent.py

```diff
@@ -2279,6 +2279,67 @@ def gagent_check_user_logoff(self, test, params, env):
         error_context.context("Verify if guest agent works.", logging.info)
         self.gagent_verify(self.params, self.vm)
 
+    @error_context.context_aware
+    def gagent_check_blacklist(self, test, params, env):
+        """
+        Verify the blacklist of config file, linux guest only
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        def bl_check(qga_cmd):
+            """
+            check if qga cmd is disabled.
+            """
+            try:
+                if qga_cmd == "guest-file-open":
+                    self.gagent.guest_file_open(guest_file, mode="a+")
+                else:
+                    self.gagent.cmd(qga_cmd)
+            except guest_agent.VAgentCmdError as detail:
+                if re.search("has been disabled", str(detail)):
```

Comment:
why not search to find the keyword `qga_cmd`?

Reply:
you mean add cmd like guest-file-open ? Actually it works if not search qga_cmd, but add the cmd will be better, will update.
{"error": {"class": "GenericError", "desc": "The command guest-file-open has been disabled for this instance"}}